osCam/templates/user/add_config.html

```diff
@@ -0,0 +1,31 @@
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Add Camera Configuration</title>
+        {% load static %}
+        {% include "core/bootstrap.html" %}
+        <link rel="stylesheet" href="{% static 'core/dash.css' %}">
+
+        {% load crispy_forms_field %}
+        {% load crispy_forms_tags %}
+    </head>
+        <body>
+            {% include "core/navigation.html" %}
+            <div class="Jumbotron"
+                style="color: rgba(15, 34, 75, 0.644); background: beige; margin-left: 30%; margin-right: 30%">
+                <h1 class="text-center">Add Camera Configuration</h1>
+                <div class='crispy_forms_field' style="margin-left: 10%; margin-right: 10%; font-size: 20pt;">
```

Comment:
this CSS will go in that file as well.., just give this DIV tag a different id or class attribute (since it has different style attributes) and reference as the example I gave shows =)